data/mods/Xedra_Evolved/npc/boann.json

```
@@ -16,7 +16,12 @@
     "name": "fae noblewoman",
     "job_description": "I'm in charge of the faeries of my court.",
     "common": false,
-    "traits": [ { "trait": "hair_long", "variant": "blond" }, { "trait": "SKIN_DARK" }, { "trait": "FAERIECREATURE" } ],
+    "traits": [
+      { "trait": "hair_long", "variant": "blond" },
+      { "trait": "natural_hair_color_blond" },
```

```suggestion
      { "trait": "natural_hair_color_blond" },
```
So does this line mean the npc’s hair will grow over time now or what exactly does this do?

---

no, the trait that respond for hair grow is `natural_hair_growth`, this one seems to be a blank trait if character has no hair color defined

---

This trait assigns a natural hair color to the NPC in question, so that when their hair grows, it will be their natural color. At this time, NPCs cosmetic traits will not change on their own, because they lack the hair growth trait (the one that players use to opt into the system). I assigned the NPCs these traits because I have plans for a followup PR which will add the code needed for NPCs to maintain a style on their own.